xchange-ascendex/src/main/java/org/knowm/xchange/ascendex/dto/balance/AscendexBalanceTransferRequestPayload.java

```diff
@@ -0,0 +1,26 @@
+package org.knowm.xchange.ascendex.dto.balance;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.knowm.xchange.ascendex.dto.enums.AccountCategory;
+
+/**
+ * Package:org.knowm.xchange.ascendex.dto.account
+ * Description:
+ *
+ * @date:2022/7/16 20:20
+ * @author:wodepig
+ */
+@Data
+@AllArgsConstructor
+public class AscendexBalanceTransferRequestPayload {
+
+    String amount;
+    String asset;
+    AccountCategory fromAccount;
+    AccountCategory toAccount;
+
+    public void setAsset(String asset) {
+        this.asset =asset==null?null: asset.toUpperCase();
```

Comment:
mapping should be done in `*Adapter` classes, not in dtos

dtos should represent what came from the API